vnext/Microsoft.ReactNative/ReactHost/JSCallInvokerScheduler.h

```diff
@@ -19,6 +19,7 @@ struct IJSCallInvokerQueueScheduler : IUnknown {
 };
 
 Mso::CntPtr<IDispatchQueueScheduler> MakeJSCallInvokerScheduler(
+    winrt::Microsoft::ReactNative::IReactNotificationService notificationService,
```

Comment:
>IReactNotificationService [](http://example.com/codeflow?start=35&length=25)

As a general rule in our code we do not pass by value types that have non-trivial destructors. These types must be passed as const& or && depending on the intended ownership: const& no or optional ownership transfer, and && when the ownership is accepted by API.

Reply:
Changed to pass by const &